deploy/operator.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  name: portworx-operator
+  namespace: kube-system
```

Comment:
Doesn't the catalog allow installing the operator in any namespace? We should not hard code this to kube-system.

Reply:
This namespace is not taken by the catalog. The catalog csv has a placeholder namespace. It takes only the pod template from this deployment.